src/crimson/os/seastore/transaction.h

```diff
@@ -190,11 +190,11 @@ class Transaction {
     ref->set_paddr(final_addr);
     assert(!ref->get_paddr().is_null());
     assert(!ref->is_inline());
-    ool_block_list.push_back(ref);
+    written_ool_block_list.push_back(ref);
     write_set.insert(*ref);
   }
 
-  void mark_extent_ool(LogicalCachedExtentRef& ref, paddr_t final_addr) {
+  void mark_allocated_extent_ool(LogicalCachedExtentRef& ref, paddr_t final_addr) {
```

Comment:
Should not pass the final_paddr in, as the addr is not decided at this step.

Instead, assert that the extent's paddr is already absolute.